build-setup/clion/using_clion.md

```diff
@@ -1,13 +1,9 @@
 # Using CLion with `ml-cpp`
 
```

Comment:
I think we should say which version of CLion you need.

I still had CLion 2020 on my machine and it didn't import our CMake project because its embedded CMake was version 3.17, which was too old.

After I upgraded to CLion 2022 everything is fine, as it comes with CMake 3.22 embedded.

I don't know about CLion 2021.  If you still have it and it works then we can say the minimum requirement is CLion 2021.  Or if you can quickly find out in a Google search which version of CMake CLion 2021 had then again, if it's 3.19.2 or above we can say 2021 is the minimum.

Otherwise, let's say CLion 2022 is the minimum required. It's not worth spending ages checking CLion 2021 if that would be time consuming.